packages/contracts/contracts/deploy/PayableDAppFactory.sol

```diff
@@ -50,6 +56,16 @@ contract PayableDAppFactory is
         transferOwnership(_owner);
     }
 
+    /// @notice sets a name for the pool using ENS service
+    function setName(string memory name) external override onlyOwner {
+        ReverseRegistrar ensReverseRegistrar = ReverseRegistrar(
+            ens.owner(ADDR_REVERSE_NODE)
+        );
+
+        // call the ENS reverse registrar resolving address to name
```

Comment:
I think "to resolve" sounds better, no?

```suggestion
        // call the ENS reverse registrar to resolve address to name
```

Reply:
Actually I think "to register name of address" makes more sense.

Reply:
Yes, this works as well.